example/dotenv

```diff
@@ -22,6 +22,7 @@ USE_LOCALSTACK=true
 SHARED_SECRET_KEY=_THIS_IS_MY_32_CHARS_SECRET_KEY_
 
 # Set to 'true' if running Docker in Docker and the docker daemon is somewhere other than the current context
+# Leave blank, i.e. DIND= if not running Docker in Docker
```

Comment:
This caught me out a little when I started working with this service, so the extra comment is great! I think most developers (not that this is a large number) want to start with `DIND=`. Maybe it would be good to make that the default value on the line below?

_Another_ future thing - do we need the `DIND` option at all?